Mathlib/Geometry/Manifold/Algebra/SmoothFunctions.lean

```diff
@@ -282,12 +282,12 @@ def C : 𝕜 →+* C^n⟮I, N; 𝓘(𝕜, A), A⟯ where
   map_zero' := by ext; exact (algebraMap 𝕜 A).map_zero
   map_add' c₁ c₂ := by ext; exact (algebraMap 𝕜 A).map_add _ _
 
-instance algebra : Algebra 𝕜 C^n⟮I, N; 𝓘(𝕜, A), A⟯ :=
-  { --ContMDiffMap.semiring with -- Porting note: Commented this out.
-    smul := fun r f => ⟨r • f, contMDiff_const.smul f.contMDiff⟩
-    toRingHom := ContMDiffMap.C
-    commutes' := fun c f => by ext x; exact Algebra.commutes' _ _
-    smul_def' := fun c f => by ext x; exact Algebra.smul_def' _ _ }
+instance algebra : Algebra 𝕜 C^n⟮I, N; 𝓘(𝕜, A), A⟯ where
+  -- __ := ContMDiffMap.semiring with -- Porting note: Commented this out.
```

Comment:
This porting note is kind of a lie now, maybe it's time to just remove it? It's now about a structure which no longer exists.

Reply:
I agree the porting note can be removed, but I am not sure what the structure which no longer exists refers to: `ContMDiffMap.semiring` still seems to exist...